README.md

```diff
@@ -326,17 +326,39 @@ composer {
 
 ## Tolerance
 
-Shot provides a simple mechanism to be able to configure a threshold value when comparing recorded images with the new ones during the verification stage. You may need to use tolerance in your tests when testing compose components because the API Shot uses to record screenshots depending on the device where your tests are executed. There are other scenarios where you may need to configure a tolerance value, but these are not so common. If you want to configure it you can use this config in your ``build.gradle`` file.
+Shot provides a simple mechanism to be able to configure a threshold value when comparing recorded images with the new ones during the verification stage. You may need to use tolerance in your tests when testing compose components, because the API Shot uses to record screenshots depending on the device where your tests are executed, or when recording and verifying the images in different architectures (x86_64 and arm64). There are other scenarios where you may need to configure a tolerance value, but these are not so common.
+
+Be careful when using these parameters because it could make your tests less reliable.
+
+### Global tolerance
+
+This is global tolerance that allows having a percentage of pixels different from the recorded image.
+
+If you want to configure it you can use this config in your ``build.gradle`` file:
 
 ```gradle
 shot {
-    tolerance =  0.1 // 0,1% tolerance
+    tolerance =  0.1 // 0.1% tolerance
 }
 ```
 
-If you configure ``tolerance = 1`` it means the tolerance threshold will be 100% and all your tests will pass even if they should fail...so be careful when configuring this param. 
+If you configure ``tolerance = 100`` it means the tolerance threshold will be 100% and all your tests will pass even if they should fail...so be careful when configuring this param. 
```

Comment:
Not sure about it 🤔 can you please check the numbers again in the implementation and ensure this is the correct value? I do not remember if 1 == 100% or 100 = 100% tolerance and I think I'm biased because I implemented the original version. Thanks!